tardis/io/config_reader.py

```diff
@@ -1031,19 +1031,19 @@ def from_config_dict(cls, config_dict, atom_data=None, test_parser=False,
         black_body_section = montecarlo_section['black_body_sampling']
         montecarlo_section['black_body_sampling'] = {}
         montecarlo_section['black_body_sampling']['start'] = \
-            black_body_section[0]
+            black_body_section['start']
         montecarlo_section['black_body_sampling']['end'] = \
-            black_body_section[1]
+            black_body_section['stop']
         montecarlo_section['black_body_sampling']['samples'] = \
-            black_body_section[2]
+            black_body_section['num']
         virtual_spectrum_section = montecarlo_section['virtual_spectrum_range']
         montecarlo_section['virtual_spectrum_range'] = {}
         montecarlo_section['virtual_spectrum_range']['start'] = \
-            virtual_spectrum_section[0]
+            virtual_spectrum_section['start']
         montecarlo_section['virtual_spectrum_range']['end'] = \
-            virtual_spectrum_section[1]
+            virtual_spectrum_section['stop']
         montecarlo_section['virtual_spectrum_range']['samples'] = \
-            virtual_spectrum_section[2]
```

Comment:
What's the reason we keep this?
I propose to delete lines 1031-1046 and rename `montecarlo.virtual_spectrum_range.samples` to `montecarlo.virtual_spectrum_range.num` wherever it is accessed